app/controllers/users/sessions_controller.rb

```diff
@@ -204,6 +205,32 @@ def override_csp_for_google_analytics
     def sign_in_params
       params[resource_name]&.permit(:email) if request.post?
     end
+
+    def check_password_compromised
+      return if current_user.check_password_compromised_at.present? ||
```

Comment:
Maybe fine to save for later as long we create a ticket to track, but: I assume we'll want to check someone's password more than once ever, so we should decide how often we'll want to do that.

Reply:
Yea I was thinking this is just the initial run, I wanted to leave space for a check after a certain timeframe. or after a user changes their password.

Reply:
created https://cm-jira.usa.gov/browse/LG-13063 to look into this